optaplanner-spring-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/OptaPlannerAutoConfiguration.java

```diff
@@ -180,19 +180,8 @@ public SolverConfig solverConfig() {
                     }
                 };
             }
-            ConstraintProvider_ constraintProvider = ConfigUtils.newInstance(this, "constraintProvider",
-                    (Class<ConstraintProvider_>) solverConfig.getScoreDirectorFactoryConfig().getConstraintProviderClass());
-            Class<?>[] entityClasses = solverConfig.getEntityClassList().toArray(new Class<?>[0]);
-            ConstraintStreamImplType constraintStreamImplType =
-                    solverConfig.getScoreDirectorFactoryConfig().getConstraintStreamImplType();
-            boolean droolsAlphaNetworkCompilationEnabled =
-                    solverConfig.getScoreDirectorFactoryConfig().isDroolsAlphaNetworkCompilationEnabled();
-
-            return (ConstraintVerifier<ConstraintProvider_, SolutionClass_>) ConstraintVerifier.build(constraintProvider,
-                    solverConfig.getSolutionClass(), entityClasses)
-                    .withConstraintStreamImplType(
-                            (constraintStreamImplType != null) ? constraintStreamImplType : ConstraintStreamImplType.DROOLS)
```

Comment:
This mention of the default (DROOLS) in the spring starter is what I really wanted to get rid of, as it belongs in the ConstarintVerifier.

Reply:
If you don't want to pick a default, don't call the method - default will be picked for you.